src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
+/*
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PriseDay = 40;
+
+  if (days < 3) {
+    return days * PriseDay;
+  } else if (days >= 3 && days < 7) {
+    return days * PriseDay - 20;
+  } else {
+    return days * PriseDay - 50;
```

Comment:
Replace hard-coded values with constants for better readability and maintainability. For example, use `SHORT_TERM` and `SHORT_TERM_DISCOUNT` for 3 days and $20 off, and `LONG_TERM` and `LONG_TERM_DISCOUNT` for 7 days and $50 off.